added url to PHP documention datetime.format (#61)
TYPO3-Documentation/TYPO3CMS-Reference-ViewHelper: Documentation/typo3/fluid/latest/Format/Date.rst

   false
:aspect:`Description`

>    Format String which is taken to format the Date/Time
>    
>    A documentation with possible values can be found here: https://www.php.net/manual/de/datetime.format.php

Reviewer: ```suggestion
   Format string which is taken to format the date/time, see the `PHP documentation <https://www.php.net/manual/datetime.format.php>`_
```
